### PR TITLE
feat: remove uniqueness from sequelize storage

### PR DESCRIPTION
**Pull request by mau31415**
…tries in case of rerun

### CODE REVIEW

**Review by danAlmogSB — APPROVED** [review]
useful change

**Comment by mmkal** [comment]
Thanks. Could you add a test that fails without this change?

File: test/storage/sequelize.test.ts [hunk 1]
```diff
@@ -266,6 +287,19 @@ describe('sequelize', () => {
...
+				sequelize: helper.sequelize,
+			});
+
+			await storage.logMigration({ name: 'm1' });
+			expect(await storage.executed()).toEqual(['m1']);
+
+			await storage.logMigration({ name: 'm1' });
+			await storage.logMigration({ name: 'm2' });
+			expect(await storage.executed()).toEqual(['m1', 'm1', 'm2']);
+		});
```
Comment (mmkal) [inline test/storage/sequelize.test.ts:302]: Thank you for adding this test, it makes it clearer. Are we losing some safety by making it non-unique by default though?

**Comment by jbalatero** [comment]
Since this was reverted, how do we work around --rerun ALLOW now?

**Comment by mau31415** [comment]
I solved it on my end, I didn't have time to create a better fix. Sorry...

**Pull request merged**